src/sass/components/_section--hot-deals.scss

```diff
@@ -0,0 +1,263 @@
+.cart {
+  box-sizing: border-box;
+
+  &-title {
+    padding: 16px;
+    margin-bottom: 0;
+    background: $footer-claim-bg;
+    display: flex;
+    justify-content: space-between;
+    text-transform: uppercase;
+    color: white;
+    font-size: 20px;
+    line-height: 18px;
+
+    p {
+      margin-bottom: 0;
+    }
+
+    .col-auto {
+      padding: 0;
+    }
+
+    .dots {
+      ul {
+        margin: 0;
+        padding: 0px;
+        list-style: none;
+        background-color: $footer-claim-bg;
+
+        li {
+          display: inline-block;
+          margin-left: 0.5rem;
+
+          a {
+            display: block;
+            width: 13px;
+            height: 13px;
+            border-radius: 6px;
+            background-color: #e1e1e1;
+
+            &.active,
+            &:hover {
+              background-color: $primary;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .cart-body {
+    position: relative;
+    background: gray;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .product-photo {
+      width: 100%;
+      height: auto;
+    }
+
+    .add-to-cart {
+      cursor: pointer;
+      position: absolute;
+      background: $footer-claim-bg;
+      text-transform: uppercase;
+      display: flex;
+      padding: 8px;
+      left: 50%;
+      bottom: 50%;
+      transform: translate(-50%);
+      color: white;
+
+      p {
+        margin-left: 5px;
+        margin-bottom: 0;
+        font-size: 13px;
+      }
+    }
+  }
+
+  .time-count {
+    position: absolute;
+    bottom: 0;
+    display: flex;
+
+    .time {
+      background: $counter-color;
+      border-radius: 50%;
+      color: white;
+      width: 70px;
+      height: 70px;
+      font-size: 18px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      text-transform: uppercase;
+      margin: 9px 3px;
+    }
+  }
```

Comment:
Jeśli dobrze zrozumiałam treść zadania (jeśli się mylę to przepraszam), add-to-cart oraz liczniki powinny się pojawiać dopiero po najechaniu na zdjęci.

Reply:
nieuwaga :) odp. w komentarzu głównym.